doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -452,33 +465,34 @@ Induction
    arguments are correct is done only when registering the
    lemma in the global environment. To know if the use of induction hypotheses
    is correct during the interactive development of a proof, use
-   the command :cmd:`Guarded` (see Section :ref:`requestinginformation`).
+   the command :cmd:`Guarded`.
 
    :n:`with {+ ( @ident {* @simple_binder } {? %{ struct @name %} } : @type ) }`
      Starts a proof by mutual induction. The statements to be proven
      are :n:`forall @simple_binder__i, @type__i`.
-     The identifiers :n:`@ident` are the names of the induction hypotheses. The identifiers
-     :n:`@ident` are the respective names of the premises on which the induction
+     The identifiers :n:`@ident` (including the first one before the `with` clause)
+     are the names of the induction hypotheses. The identifiers
+     :n:`@name` are the respective names of the premises on which the induction
      is performed in the statements to be proved (if not given, Coq
      guesses what they are).
 
 .. tacn:: cofix @ident {? with {+ ( @ident {* @simple_binder } : @type ) } }
 
-   .. todo:
-
-   Starts a proof by co-induction. The first :n:`@ident` is the
-   name of the co-induction hypothesis. As in a cofix expression,
+   Starts a proof by co-induction. The :n:`@ident`\s (including the first one
```

Comment:
There's a lack of consistency in the description of `fix` and `cofix`. Either the main description should focus on the case without the `with` clause and `ident` should be then singular (in both cases) or it should already encompass the optional use of the `with` clause and then it should be plural in both cases (possibly with the `s` in parentheses).